SwiftWisdom/Rx/Rx+Extensions.swift

```diff
@@ -47,6 +47,22 @@ public func <- <T>(observer: Variable<T>, observable: Variable<T>) -> Disposable
     return observer <- observable.asObservable()
 }
 
+public func <- <T: ObserverType, O>(observer: T, behaviorRelay: BehaviorRelay<O>) -> Disposable where T.E == O {
+    return observer <- behaviorRelay.asObservable()
+}
+
+public func <- <T: ObserverType, O>(observer: T, behaviorRelay: BehaviorRelay<O>) -> Disposable where T.E == O? {
+    return observer <- behaviorRelay.asObservable()
+}
+
+public func <- <T, O: ObservableType>(behaviorRelay: BehaviorRelay<T>, observable: O) -> Disposable where O.E == T {
```

Comment:
out of curiosity, why is the order of params different on this function?

Reply:
Not sure there is one. Just following the prior convention In 'Variable' binding of placing the argument with a child first.